docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,467 @@
+  in the tx, the interface should be called +
+  - the creation of a new signing group should be triggered (only sometimes, if
+  it’s the right time).
+
+| Relay request - relay entry submission
+| The transaction submitting relay entry is not reimbursable (i.e. the submitter
+  must provide the fee required for its processing).
+
+| Relay request - relay entry submission
+| The transaction submitting relay entry should have a predictable cost and the
+  gas cost of this transaction should be as low as possible, below 200k gas.
+
+| Relay request - relay entry submission
+| It should be possible to send low gas callbacks (up to the fixed gas cost
+  value) in the transaction submitting the relay entry.
+
+| Relay request - relay entry submission
+| Failure in the callback function should not revert the relay entry transaction.
+
+| Relay request - relay entry submission
+| The operation removing members from the sortition pool should be as cheap as
+  possible so that the member submitting relay entry does not have to pay any
+  additional costs for removing inactive members from the pool.
+
+| Relay request - relay entry timeouts
+| The soft relay entry timeout equals the group size multiplied by the number of
+  blocks for a member to become eligible to submit relay entry. The timer starts
+  at the moment when the first member becomes eligible (the moment when the
+  relay entry was requested). +
+  The hard relay entry timeout is a set value (measured in time units).
+
+| Relay request - relay entry timeouts
+| If no entry was successfully submitted until soft relay timeout is reached,
+  all operators in the group should start bleeding and losing their T stake. The
+  amount of lost stake should incrementally increase from 0 (at the moment of
+  soft relay entry timeout) to a fixed value (at the moment of hard relay entry
+  timeout).
+
+| Relay request - relay entry timeouts
+| If no entry was successfully submitted until soft relay timeout is reached,
+  all operators in the group should start bleeding and losing their T stake. The
+  amount of lost stake should incrementally increase from 0 (at the moment of
+  soft relay entry timeout) to a fixed value (at the moment of hard relay entry
+  timeout).
+
+| Relay request - relay entry timeouts
+| Anyone can send a transaction reporting hard relay entry timeout.
+
+| Relay request - relay entry timeouts
+| Upon noticing the hard relay entry timeout notification, the random beacon
+  contract should determine its validity.
+
+| Relay request - relay entry timeouts
+| When determining validity of the hard relay entry timeout notification, the
+  following aspects should be taken into account: +
+  - does it reference the existing relay entry request? (if not, then
+  notification invalid) +
+  - has the hard relay entry timeout passed? (if not, then notification invalid) +
+
+| Relay request - relay entry timeouts
+| If the hard relay entry timeout notification was deemed justified, then: +
```

Comment:
Also:
- All members of the terminated group should be slashed up to the fixed slashing amount
- If there are no active groups, relay entry could not be repeated. The request should be cleaned up to allow new requests to happen.

Reply:
Added, see 953ccbb65131b597e4fe3bba5ac4dca810aadcf1 and 3cb023632eeaf6e1143d26413657da592d2b99aa.